app/common/lib/ledgerUtil.js

```diff
@@ -18,49 +18,24 @@ const settings = require('../../../js/constants/settings')
 // Utils
 const {responseHasContent} = require('./httpUtil')
 const urlUtil = require('../../../js/lib/urlutil')
-const {makeImmutable} = require('../state/immutableUtil')
 const getSetting = require('../../../js/settings').getSetting
 
 /**
  * Is page an actual page being viewed by the user? (not an error page, etc)
  * If the page is invalid, we don't want to collect usage info.
- * @param {Map} view - an entry from ['pageData', 'view']
- * @param {List} responseList - full ['pageData', 'load'] List
+ * @param {Map} tab - data about provided tab
  * @return {boolean} true if page should have usage collected, false if not
  */
-const shouldTrackView = (view, responseList) => {
-  if (view == null) {
+const shouldTrackView = (tab) => {
```

Comment:
nit: please rename this to tab because we usually mean tab as the webContents in code.

Reply:
done